EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -169,7 +169,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
-		4C2DEF55201B8FAD0062315E /* libSDL2-2.0.0.dylib */ = {isa = PBXFileReference; lastKnownFileType = "compiled.mach-o.dylib"; name = "libSDL2-2.0.0.dylib"; path = "/usr/local/Cellar/sdl2/2.0.8/lib/libSDL2-2.0.0.dylib"; sourceTree = "<absolute>"; };
```

Comment:
This change does not seem intentional and may in fact result in breakage.

Reply:
This is what I needed to change to get this to build in XCode on my computer. I installed through brew. This is the same location referenced in the Wiki for Build Instructions Library Paths. https://github.com/endless-sky/endless-sky/wiki/BuildInstructions

Reply:
@nobodywasishere please also test if you can create a fully distributable Mac build per those instructions. This is an issue we've been dealing with on the Discord. If you like, you may join us there at discord dot gg slash ZeuASSx.

Reply:
Fully distributable MacOS build can be made still.